README.md

```diff
@@ -72,7 +72,7 @@ Courses can be retrieved from the API:
 'Test Course'
 
 # Update the course's name
->>> course.update(name='New Course Name')
+>>> course.update(course={name: 'New Course Name'})
```

Comment:
I wish Python let us name keys without quotes.

`NameError: name 'name' is not defined`.